<?xml version="1.0"?>
<launch>
  <arg name="mav_name" default="spedix"/>
  <group ns="$(arg mav_name)">
    <!-- node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="log" >
      <param name="video_device" value="/dev/video0" />
      <param name="image_width" value="640" />
      <param name="image_height" value="480" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam" />
      <param name="io_method" value="mmap"/>
    </node -->

    <node name="detector" pkg="detector" type="detector_node" output="screen">
      <param name="mav_name" value="$(arg mav_name)"/>
      <rosparam file="$(find detector)/cfg/detector.yaml"/>
      <rosparam file="$(env HOME)/.ros/camera_info/mv.yaml"/>
      <remap from="image" to="/mv/image_raw"/>
      <remap from="odometry" to="odom"/>
      <remap from="bounding_boxes" to="bounding_boxes"/>
      <remap from="object_poses" to="object_poses"/>
      <remap from="undist_image" to="undist_image"/>
      <remap from="marked_image" to="marked_image"/>
      <remap from="buffer" to="buffer"/>
    </node>
  </group>
</launch>
